From 167d5e5d5f392017ae56fbedc909f098e36019f4 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Wed, 21 Nov 2018 16:34:50 +0200
Subject: [PATCH 5177/5242] MLK-20328-5: ARM64: dtsi: mx8mm: adjust the
 expected format

commit  749b10a9e12ac637afdad7c4a17fbd3547472843 from
https://source.codeaurora.org/external/imx/linux-imx.git

Default value must be prefixed by "0".

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit b81a9884f85f750673a4ee0da60096d330add7a5)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
index 3f5144a..d3669ab 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm.dtsi
@@ -724,7 +724,7 @@
 		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dmas = <&sdma2 0 2 0>, <&sdma2 1 2 0>;
 		dma-names = "rx", "tx";
-		fsl,dataline = <0xff 0xff>;
+		fsl,dataline = <0 0xff 0xff>;
 		status = "disabled";
 	};
 
@@ -769,7 +769,7 @@
 		dmas = <&sdma2 8 2 0>, <&sdma2 9 2 0>;
 		dma-names = "rx", "tx";
 		fsl,shared-interrupt;
-		fsl,dataline = <0xf 0xf>;
+		fsl,dataline = <0 0xf 0xf>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

